spec/std/spec/cli_spec.cr

```diff
@@ -0,0 +1,158 @@
+require "spec"
+require "spec/cli"
+
+private class SpecRunnerCLI < Spec::CLI
+  getter exit_code : Int32?
+
+  private def display(message)
```

Comment:
This mock shouldn't override `display` and `terminate` but `exit`. If you define it as a private instance method, it shadows the top level `::exit`.